### PR TITLE
finagle-partition: -> 2.13

### PR DESCRIPTION
**Pull request by martijnhoekstra**
Problem

No 2.13 build for finagle-partition

Solution

add 2.13 to the cross-build targets, do import renames and eliminate a breakout

Result

### CODE REVIEW

File: finagle-partitioning/src/main/scala/com/twitter/finagle/partitioning/KetamaFailureAccrualFactory.scala [hunk 1]
```diff
@@ -6,7 +6,8 @@ import com.twitter.finagle.client.Transporter
 import com.twitter.finagle.liveness.{FailureAccrualFactory, FailureAccrualPolicy}
 import com.twitter.finagle.service.{ReqRep, ResponseClassifier}
 import com.twitter.finagle.stats.StatsReceiver
-import com.twitter.finagle._
+import com.twitter.finagle.{param => finagleparam, _}
```
Comment (yufangong) [inline finagle-partitioning/src/main/scala/com/twitter/finagle/partitioning/KetamaFailureAccrualFactory.scala]: nits: I checked a few files, import alias usually being camel cases.

Comment (martijnhoekstra) [reply finagle-partitioning/src/main/scala/com/twitter/finagle/partitioning/KetamaFailureAccrualFactory.scala]: I'll get to that tomorrow.

Comment (martijnhoekstra) [reply finagle-partitioning/src/main/scala/com/twitter/finagle/partitioning/KetamaFailureAccrualFactory.scala]: Narrator: he didn't.

Comment (martijnhoekstra) [reply finagle-partitioning/src/main/scala/com/twitter/finagle/partitioning/KetamaFailureAccrualFactory.scala]: Did now though

**Comment by yufangong** [comment]
Merged here https://github.com/twitter/finagle/commit/89f06885140ee186f7b8d7ff15169a7a3f8b676b
TY!

**Pull request closed without merging**